api/draft_registrations/views.py

```diff
@@ -49,7 +50,7 @@ def check_resource_permissions(self, resource):
 
 class DraftRegistrationList(NodeDraftRegistrationsList):
     permission_classes = (
-        IsContributorOrAdminContributor,
+        IsAdminContributor,
```

Comment:
Just a note to ensure that the permissions for the draft registration api endpoints are in line with docs/expectations. I don't remember what it was about specifically but there was some back-and-forth on what exactly the permissions levels should be on these endpoints.

Reply:
I think ContributorOrAdmin is correct, though the current front end will only allow Admin Contributors to view. We are not yet changing the behavior of the existing front end.

Reply:
I changed it to `IsAdminContributor` to match the `NodeDraftRegistrationsList` permission classes, which include IsAdminContributor. Which is right?